py/picca/io.py

```diff
@@ -962,7 +970,14 @@ def read_from_desi(in_dir, catalog, pk1d=None):
     if 'TARGETID' in catalog.colnames:
         id_name = 'TARGETID'
         plate_name = 'TILEID'
-        mjd_name = 'NIGHT'
+        if 'NIGHT' in catalog:
+            mjd_name = 'NIGHT'
+        elif 'FIRST_NIGHT' in catalog:
+            mjd_name = 'FIRST_NIGHT'
+        else:
+            print("night info not stored in catalog, will replace it with 'unknown' and continue")
+            mjd_name='NIGHT'
+            catalog['NIGHT']='unknown'
```

Comment:
Is this supposed to be added to read_from_minisv_desi too? I get a key error for entry['NIGHT'] on line 1274.

Reply:
I just did that now, thanks for the info.

Reply:
Works for me now, cheers.